Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Use the official Rust image as the base image
+FROM rust:1.81 AS builder
+
+# Install protobuf compiler
+RUN apt-get update && apt-get install -y protobuf-compiler git
```

Comment:
Probably we would like to to clean after these update / install as the runtime dependencies